Content.Server/Fax/FaxSystem.cs

```diff
@@ -449,6 +456,10 @@ public void Copy(EntityUid uid, FaxMachineComponent? component = null)
         // will start immediately.
 
         UpdateUserInterface(uid, component);
+
+        if (args.Session.AttachedEntity != null)
+            _adminLogger.Add(LogType.Action, LogImpact.Low,
+                $"{ToPrettyString(args.Session.AttachedEntity.Value):actor} added copy job to {ToPrettyString(uid):tool} with text: {ToPrettyString(component.PaperSlot.Item):subject}");
```

Comment:
`ToPrettyString(component.PaperSlot.Item):subject` does not print the contents; `paper.Content` does.